templates/documentation/work-and-careers.html

```diff
@@ -39,82 +39,132 @@
+        <div class="col-6 col-medium-4">
+          <p>Candidates who progress will face up to three interviews with Technical Author peers, covering:</p>
+          <ul class="p-list--divided">
+            <li class="p-list__item has-bullet"><em>Skills and experience</em>: What have you done? What do you know? What can you do?</li>
+            <li class="p-list__item has-bullet"><em>Insight and understanding</em>: How do you think about documentation and its challenges? What ideas have you formed, about practice, process and product? How deep is your thinking?</li>
+            <li class="p-list__item has-bullet"><em>Culture and value</em>: What drives you and what values underpin your work? How do you collaborate? How will you fit in with our values, and how will you challenge us to do better?</li>
           </ul>
         </div>
+        <div class="col-9 col-medium-6">
+          <hr class="p-rule">
+        </div>
+        <h3 class="col-3 col-medium-2 p-heading--5">Talent interview</h3>
+        <div class="col-6 col-medium-4">
+         <p>Next is an interview with someone from our Talent Acquisition team, that looks back at your career, discusses how you work and want to work, and considers questions like salary, availability to start and so on.</p>
+        </div>
+        <div class="col-9 col-medium-6">
+          <hr class="p-rule">
+        </div>
+        <h3 class="col-3 col-medium-2 p-heading--5">Technical exercise</h3>
+        <div class="col-6 col-medium-4">
+          <p>Not strictly an interview, but an opportunity to demonstrate your ability to think through and solve documentation problems (and it's not a writing test; we already know you can write). We want to see how you approach issues such as meeting user needs and documentation structure, and your analytical and critical skills.</p>
+        </div>
+        <div class="col-9 col-medium-6">
+          <hr class="p-rule">
+        </div>
+        <h3 class="col-3 col-medium-2 p-heading--5">Documentation lead interview</h3>
+        <div class="col-6 col-medium-4">
+          <p>An interview to confirm our assessments and work out the next steps, based on your interests, skills and experience. This will help us arrange interviews with appropriate hiring managers in different teams.</p>
+        </div>
+        <div class="col-9 col-medium-6">
+          <hr class="p-rule">
+        </div>
+        <h3 class="col-3 col-medium-2 p-heading--5">Hiring manager interviews</h3>
+        <div class="col-6 col-medium-4">
+          <p>By this stage, we'd like to have you as a Technical Author at Canonical, and now it's a question of finding the right team or project for you, where both we and you feel that you'll be excited by the technology and in a strong position to make a valuable contribution.</p>
+          <p>That is indeed a lot of interviews. It's an in-depth process for an exacting role and we take it very seriously.</p>
```

Comment:
I'm confused about this bit and I really wish we had an updated mock-up to work with. On the live site this is separated by an hr, and on the old mock-up it was handled like this:
![image](https://github.com/canonical/canonical.com/assets/17607612/a2c7f391-1932-4ea6-90da-1672972fcb32)

But the new design doesn't seem to make a distinction between this section summary and and the "hiring manager interviews" portion. I think we should at least add an hr, what do you think?